matplotlibrc.template

```diff
@@ -416,6 +416,11 @@
 #polaraxes.grid : True  ## display grid on polar axes
 #axes3d.grid    : True  ## display grid on 3d axes
 
+## ***************************************************************************
+## * AXIS                                                      *
+## ***************************************************************************
+#axis.titlelocation : center  ## alignment of the title: {left, right, center}
```

Comment:
I would probably call this `xaxis.labellocation`/`yaxis.labellocation` (should they be separate? or together? (`axes.labellocation`?).

Reply:
Fine by me to split it (x/y/colorbar), but it seems there's opposition to having a lot of things in rcParams, so I figured this was an ok compromise

Reply:
See below re: separate rcparams. I guess colorbar can just use the value corresponding to its orientation?